OpenUpSA/wazimap-ng-ui
# Share url implementation

## Description

URL updates when filters are changed.

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-638

## How is it tested automatically?


## How should a reviewer test it locally
Changing filters should update url
Copy pasting that url should apply same filters
applied filters should come in my view
Pressing forward and backward on browser should apply/remove filters
url with geography should open up specific geography with filters applied


## Screenshots


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
